content/en/docs/installation/guide/operator-k3s.md

```diff
@@ -0,0 +1,297 @@
+
+You can find detailed configuration information in the [Armory Operator Configuration]({{< ref "operator-config" >}}) guide.
+
+### Update Armory version and S3 bucket name
+
+Update the `spec.spinnakerConfig.config.version` value to the version of Armory you want to deploy. Check the [Release Notes]({{< ref "rn-armory-spinnaker" >}}) if you are unsure which version to install. Choose v2.20.4, v2.20.5 or v2.21+ if you want to deploy plugins.
+
+Additionally, replace **myBucket** (`spec.spinnakerConfig.config.persistentStorage.s3.bucket`) with the name of the s3 bucket you created in [Create an S3 Bucket](#create-an-s3-bucket).
+
+Before editing:
+
+```yaml
+kind: SpinnakerService
+metadata:
+  name: spinnaker
+spec:
+  # spec.spinnakerConfig - This section is how to specify configuration spinnaker
+  spinnakerConfig:
+    # spec.spinnakerConfig.config - This section contains the contents of a deployment found in a halconfig .deploymentConfigurations[0]
+    config:
+      version: 2.15.1   # the version of Spinnaker to be deployed
+      persistentStorage:
+        persistentStoreType: s3
+        s3:
+          bucket: mybucket
+          rootFolder: front50
+```
+
+For example, after editing if you named your bucket `my-s3-bucket` and want to install Armory 2.21.1:
+
+```yaml
+kind: SpinnakerService
+metadata:
+  name: spinnaker
+spec:
+  # spec.spinnakerConfig - This section is how to specify configuration spinnaker
+  spinnakerConfig:
+    # spec.spinnakerConfig.config - This section contains the contents of a deployment found in a halconfig .deploymentConfigurations[0]
+    config:
+      version: 2.21.1   # the version of Spinnaker to be deployed
+      persistentStorage:
+        persistentStoreType: s3
+        s3:
+          bucket: my-s3-bucket
+          rootFolder: front50
+```
+
+### Modifications for running on K3s
+
+Create a `spec.spinnakerConfig.config.security` section like the example below, replacing `<your-vm-ip>` with the public IP address of your EC2 instance. The `security` section is at the same level as the `persistentStorage` section.
+
+```yaml
+kind: SpinnakerService
+metadata:
+  name: spinnaker
+spec:
+  # spec.spinnakerConfig - This section is how to specify configuration spinnaker
+  spinnakerConfig:
+    # spec.spinnakerConfig.config - This section contains the contents of a deployment found in a halconfig .deploymentConfigurations[0]
+    config:
+      version: 2.21.1   # the version of Spinnaker to be deployed
+      persistentStorage:
+        persistentStoreType: s3
+        s3:
+          bucket: my-s3-bucket
+          rootFolder: front50
+      security:
+        apiSecurity: # Gate
+          overrideBaseUrl: <your-vm-ip>:8084
+        uiSecurity: # Deck
+          overrideBaseUrl: <your-vm-ip>:9000
+```
+
+Find the `expose.service.overrides` section at the bottom of the file. Add configuration for Deck and Gate.
```

Comment:
What're your thoughts on embedding a full (but hidden/collapsed) example of the YAML file? Or linking to one. Could embed directly in the markdown or use the Github "Gist" account we created.